### ISSUE TITLE
⌨ Inline buttons do not have sufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6761]
## ⌨ Accessibility Issue

When our button component has `mode="inline"` set and either `color="quinary"` or `color="warning"` they do not have sufficient colour contrast when compared to a white background.

### WCAG 2.1 Criteria

[Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI)

## 🦋 Expected Behaviour

Inline buttons with the colour `quinary` and `warning` should be used slightly darker shades.

## 📋 Steps to Reproduce

1. Open the  "Inline button" story
2. Notice the `quinary` and `warning` buttons have lower than expect contrast against the background

## 📸 Screenshot

![Screenshot 2023-05-31 091417](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/0ca81b7e-188d-4efe-b6ae-ac3ac07f7983)

## 🙋‍♀️ Proposed Solution

- [ ] Update `inline` `quinary` to use `quinary.darker`
- [ ] Update `inline` `warning` to use `warning.darker`

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Both `quinary` and `warning` inline buttons have been updated with darker text colour

**Comment by esizer** [comment]
Fixed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6813
